### PR TITLE
Selected search filter UI/UX improvement

### PR DESCRIPTION
**Pull request by zphingphong**
https://github.com/participedia/api/issues/825

### CODE REVIEW

**Comment by ascott** [comment]
this is looking great @zphingphong, thanks! just a few small comments to address. can you please add screenshots and a gif of the interactions please? thanks!

**Comment by zphingphong** [comment]
image

image


https://www.screencast.com/t/WxZ607ffWSq

File: api/helpers/handlebars-helpers.js [hunk 1]
```diff
@@ -960,6 +960,73 @@ module.exports = {
...
   },
 
+  searchFiltersDisplay(req) {
+    // Check if selected category params exist
+    if (!req.query.hasOwnProperty('selectedCategory')) {
+      return [];
+    }
+
+    // Check if selected category value exist in record
+    var selectedCategory = req.query['selectedCategory'];
```
[the remark this answers is not in the document]
Comment (ascott) [reply api/helpers/handlebars-helpers.js]: @zphingphong sorry these comments are just coming in now, i thought these comments were posted already but they were part of the review that i had to submit separately.

**Comment by ascott** [comment]
@jesicarson the search filters design changes screenshots are above

**Comment by jesicarson** [comment]
The first one looks incorrect, the second one looks right. lets make sure this stacks properly on smaller screen sizes - do you need a mockup?

**Comment by paninee** [comment]
@jesicarson @ascott fixed the classes and the UI.

image

**Pull request merged**